crates/fuel-streams-publisher/src/main.rs

```diff
@@ -1,188 +1,64 @@
-//! This binary subscribes to events emitted from a Fuel client or node
```

Comment:
Let's keep the `main` binary in its former shape for now, and then plan around the runtime architecture for each of its processes.